examples/wifi-echo/server/esp32/main/DataModelHandler.cpp

```diff
@@ -45,7 +44,7 @@ void InitDataModelHandler()
     emAfInit();
 }
 
-void HandleDataModelMessage(System::PacketBuffer * buffer)
+void HandleDataModelMessage(const MessageHeader & header, System::PacketBuffer * buffer, SecureSessionMgrBase * mgr)
```

Comment:
this could take an Exchange (something that has to be built, might have a `SendResponse()` method on it?) instead of the raw information for where to send a response

Reply:
Right, that's the discussion that @woody-apple wants to have, I think.  ;)